surprisal/model.py

```diff
@@ -116,14 +128,18 @@ def surprise(
                 ),
                 dim=1,
             )
+            raise NotImplementedError("Attention masking needs to be implemented.")
```

Comment:
does the attn mask need to mask out the BOS token? why? I thought the model could attend to it just as well, and the default mask received from tokenizer would still work. this change would currently render using the `bos` token unusable without further implementation